spinalcordtoolbox/reports/slice.py

```diff
@@ -231,12 +229,8 @@ def _axial_center(self, image):
         centers_y = np.zeros(nz)
         for i in range(nz):
             centers_x[i], centers_y[i] = center_of_mass(data[i, :, :])
-        try:
-            Slice.nan_fill(centers_x)
-            Slice.nan_fill(centers_y)
-        except ValueError as err:
-            logger.error("Axial center of the spinal cord is not found: %s", err)
-            raise
```

Comment:
Re: eebb09e972f8041f6690ef75381b615253e7a270, it looks like this try/except has been present since the very introduction of QC reports (https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/2696f4fea7719055a33d4f843683a1716f54ccce). 

Back then, `nan_fill` looked like this:

```python
    @staticmethod
    def nan_fill(array):
        """TODO """
        array[np.isnan(array)] = np.interp(np.isnan(array).ravel().nonzero()[0],
                                           (-np.isnan(array)).ravel().nonzero()[0],
                                           array[-np.isnan(array)])
        return array
```

But was refactored in c58098932d3b563653911eb05f4e90fef5fedac2 to indeed explicitly account for the case where there are no valid values. 

So, :+1: from me for removing the try/except!